UI/PauseScreen.cpp

```diff
@@ -78,11 +78,9 @@ void AsyncImageFileView::SetFilename(std::string filename) {
 void AsyncImageFileView::Draw(UIContext &dc) {
 	using namespace Draw;
 	if (!texture_ && !textureFailed_ && !filename_.empty()) {
-		texture_ = dc.GetThin3DContext()->CreateTextureFromFile(filename_.c_str(), DETECT);
+		texture_ = CreateTextureFromFile(dc.GetThin3DContext(), filename_.c_str(), DETECT);
 		if (!texture_)
 			textureFailed_ = true;
-		else if (textureAutoGen_)
-			texture_->AutoGenMipmaps();
```

Comment:
Oh, I didn't intend to remove it, only move/redesign it, but apparently dropped the ball midway...  It'll be back.

I don't intend to actually remove any features, as D3D11 can do everything we need with a little work.